internal/cephfs/controllerserver.go

```diff
@@ -815,6 +815,17 @@ func (cs *ControllerServer) CreateSnapshot(
 			}
 		}
 
+		// Update snapshot-name/snapshot-namespace/snapshotcontent-name details on
+		// subvolume snapshot as metadata in case snapshot already exist
+		parameters := req.GetParameters()
+		if len(parameters) != 0 {
```

Comment:
this is always not empty as we have at least clusterID. the check should be for len of metadata?